numpy/core/src/_simd/_simd_inc.h.src

```diff
@@ -0,0 +1,420 @@
+/**
+ * This header works only through '_simd.dispatch.c'
+ */
```

Comment:
What do you mean by "only works through" here?

Reply:
since it contains forward declrations for private functions that defined within `_simd.dispatch.c.src`.

Reply:
perhaps all I have to do is just revert @mattip's suggestion and move back
```
#include "_simd_inc_data.h"
#include "_simd_inc_convert.h"
#include "_simd_inc_vector.h"
#include "_simd_inc_arg.h"
#include "_simd_inc_easyintrin.h"
```
to the tail of the main header `_simd_inc.h` instead of including them inside `simd_dispatch.c.src` and then removes this line.

Reply:
Which suggestion are you referring to?

Reply:
https://github.com/numpy/numpy/pull/16782#discussion_r457450839

Reply:
I moved the sub-headers back again to the tail of the main header '_simd_inc.h.src'